acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ProcessPantheonNodeRunner.java

```diff
@@ -46,14 +46,18 @@
   private final Logger PROCESS_LOG = LogManager.getLogger("tech.pegasys.pantheon.SubProcessLog");
 
   private final Map<String, Process> pantheonProcesses = new HashMap<>();
-  private final ExecutorService outputProcessorExecutor = Executors.newCachedThreadPool();
+  private ExecutorService outputProcessorExecutor = Executors.newCachedThreadPool();
 
   ProcessPantheonNodeRunner() {
     Runtime.getRuntime().addShutdownHook(new Thread(this::shutdown));
   }
 
   @Override
   public void startNode(final PantheonNode node) {
+    if (outputProcessorExecutor == null || outputProcessorExecutor.isShutdown()) {
+      outputProcessorExecutor = Executors.newCachedThreadPool();
```

Comment:
Why do we need to support creating a new executor service?  Shouldn't it only be shutdown after `close` is called?  I'd have expected `stop` to leave it running and the cluster can be restarted but `close` should be a very final operation and the cluster shouldn't be used after it's called.

Reply:
Agreed, this was a last minute addition following the similar pattern as ThreadP*NodeRunner. I will remove this call as well.